apps/condo/pages/ticket/index.tsx

```
@@ -648,97 +640,86 @@ const FiltersContainer = ({ filterMetas }) => {
+                                >
+                                    {ReturnedLabel}
+                                </Checkbox>
+                            </Col>
+                            <Col>
+                                <Checkbox
+                                    onChange={(event) => handleChangeIsCompletedAfterDeadline(event?.target?.checked || false)}
+                                    checked={isCompletedAfterDeadline}
+                                    style={CHECKBOX_STYLE}
+                                    id='ticket-filter-completed-after-deadline'
+                                    data-cy='ticket__filter-isCompletedAfterDeadline'
+                                    children={ExpiredLabel}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)